looprpc/client.proto

```diff
@@ -492,12 +516,7 @@ message QuoteResponse {
 
     /**
     An estimate of the on-chain fee that needs to be paid to sweep the HTLC for
-    a loop out or to pay to the HTLC for loop in. If a miner fee of 0 is
-    returned, it means the external_htlc flag was set for a loop in and the fee
-    estimation was skipped. If a miner fee of -1 is returned, it means lnd's
-    wallet tried to estimate the fee but was unable to create a sample
-    estimation transaction because not enough funds are available. An
-    information message should be shown to the user in this case.
+    a loop out or to pay to the HTLC for loop in.
     */
     int64 miner_fee = 3;
```

Comment:
while we're splitting these out, we could change the field names to be loop type specific? Since grpc doesn't care if we change names, and people will already have compile-time errors when they upgrade.

Something like `sweep_fee` for out, `htlc_publish_fee` for in.

Perhaps also mention that these fees are sats in the comments. I know it's assumed for on chain, but no harm being over-specific.

Reply:
Good suggestion, fixed